bzt/modules/blazemeter.py

```diff
@@ -196,6 +196,9 @@ def prepare(self):
             self._session = Session(self._user, {'id': sess_id})
             self._session['userId'] = self.parameters.get("user-id", None)
             self._session['testId'] = self.parameters.get("test-id", None)
+            data_address = self.settings.get("data-address", None)
```

Comment:
in order for it to work on additional environments (without it, it won't work on dev environments)